packages/react-codemirror/package.json

```diff
@@ -18,22 +18,14 @@
     "codemirror 6"
   ],
   "version": "2.0.0-next.4",
-  "main": "./dist/cjs/index.cjs",
-  "module": "./dist/esm/index.mjs",
-  "types": "./dist/types/index.d.ts",
-  "exports": {
-    ".": {
-      "require": "./dist/cjs/index.cjs",
-      "import": "./dist/esm/index.mjs",
-      "default": "./dist/cjs/index.cjs"
-    }
-  },
+  "main": "./src/index.ts",
+  "type": "module",
   "sideEffects": false,
   "scripts": {
-    "build": "concurrently 'npm:build-types' 'npm:build-esm' 'npm:build-commonjs'",
+    "build": "echo done",
```

Comment:
whoops

Reply:
Is this correct?

Reply:
no, this I need to correct. I only got the workerpool working in codemirror by not bundling. But we still need to build for non-typescript consumers